process/peer/validatorsProvider.go

```diff
@@ -85,6 +85,21 @@ func (vp *validatorsProvider) GetLatestValidators() map[string]*state.ValidatorA
 	return mapToReturn
 }
 
+// GetLatestValidatorInfos gets the latest configuration of validators per shard from the peerAccountsTrie
```

Comment:
Network sharding relies on the nodesCoordinator to get the validators on shard. It is better to use this instead?

Reply:
I removed the method, because it will be used in the next version. It will be used to populate a cache and have the latest version for a validator. If it sent a transaction for unstake-ing in the middle of the epoch he will appear as leaving only in trie. It will be a combination between trie and nodesCoordinator.

Reply:
ok